kedro-datasets/docs/source/index.md

```diff
@@ -1,12 +1,20 @@
-kedro_datasets
-==============
+```{include} ../../README.md
+:relative-images:
+```
 
-.. rubric:: Description
+## API documentation
 
+```{rubric} Description
+```
+
+```{eval-rst}
 .. automodule:: kedro_datasets
+```
 
```

Comment:
Maybe don't need all this as shown? Just hide them?

Reply:
In that case, let me go back to the previous structure, but with the nicer README!